javascript/react/src/components/ArqueroDetailsList/hooks/useGetColumnValidationErrors.ts

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import type { FieldError, ValidationResult } from '@datashaper/schema'
+
+export function useGetColumnValidationErrors(
+	validationResult?: ValidationResult,
+): string {
+	let message = ''
```

Comment:
All of this content should be in a useMemo or the forEach will happen on every render